code/modules/mob/living/carbon/xenomorph/Embryo.dm

```diff
@@ -89,7 +89,7 @@
 			if(stage < 5)
 				counter += 1 * hive.larva_gestation_multiplier
 
-		if(stage < 5 && counter >= 90)
+		if(stage < 5 && counter >= 84)
```

Comment:
Make this into a define please (the 84), magic numbers suck

Reply:
Beagle is moving burst timer to config so I'm happy to just delete this change

Reply:
now it should match 3844